modyn/selector/internal/selector_strategies/coreset_strategy.py

```diff
@@ -73,12 +91,20 @@ def _get_data(self) -> Iterable[list[int]]:
         assert isinstance(
             self._storage_backend, DatabaseStorageBackend
         ), "CoresetStrategy currently only supports DatabaseBackend"
+        # We set is_warmup here as well to support unit tests more easily.
+        # In end-to-end workflows, it should have been set by trigger() already.
+        if (self._next_trigger_id + 1) > self.warmup_triggers:
+            self.is_warmup = False
 
+        presampling_strategy = self.warmup_presampler if self.is_warmup else self.presampling_strategy
+        assert presampling_strategy is not None  # satisfy mypy
         trigger_dataset_size = None
-        if self.presampling_strategy.requires_trigger_dataset_size:
+        if presampling_strategy.requires_trigger_dataset_size:
             trigger_dataset_size = self._get_trigger_dataset_size()
 
-        stmt = self.presampling_strategy.get_presampling_query(
+        logger.error(trigger_dataset_size)
```

Comment:
maybe logger.info?

Reply:
actually this was leftover debugging code. thanks again for catching!